### ISSUE TITLE
Stackage nightly haddock failure

### ISSUE THREAD

**Issue by decentral1se** [lpeterse/haskell-socket#47]
```
Running Haddock for socket-0.8.0.0...
Preprocessing library socket-0.8.0.0...
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: array-0.5.1.1, base-4.9.1.0,
bytestring-0.10.8.1, deepseq-1.4.2.0, ghc-prim-0.5.0.0, integer-gmp-1.0.0.1

src/System/Socket/Internal/AddressInfo.hsc:253:3: error:
    parse error on input <E2><80><98>getAddressInfo<E2><80><99>
```

**Comment by lpeterse** [comment]
Fixed in 0.8.0.1 (is on Hackage).

**Comment by decentral1se** [comment]
Thanks!
